app/facades/quotes_facade.rb

```diff
@@ -0,0 +1,13 @@
+class QuotesFacade
+  def self.get_search_results(query)
+    search_results = QuotesService.search_quotes(query)
+    quotes = search_results[:results].map do |quote_data|
+      Quote.new(quote_data)
+    end
+    count = search_results[:totalCount]
+    {
+      count: count,
+      results: quotes
+    }
```

Comment:
This was an excellent design choice in order to avoid having to make a second API call just to get the count of results.